### ISSUE TITLE
Provide the overflowWidget as a builder

### ISSUE THREAD

**Issue by mattia-venturini** [elileo1/extended_wrap#5]
I have an use case where I need to show 2 lines of elements and then, if needed, a chip with "+N" (when N is the number of the remaining items).

It would be nice to have a constructor that takes a parameter `overflowBuilder: (int remaining) => SomeWidget(remaining)`.

**Comment by yahyaMajdAlamaal** [comment]
I have same case
Is there any solution?
@elileo1

**Comment by yahyaMajdAlamaal** [comment]
@mattia-venturini 
Did you find a solution?

**Comment by mattia-venturini** [comment]
@yahyaMajdAlamaal unfortunately no.

Later I find out that in my use case the ExtendedWrap had some important **performance issues** (maybe I should file another issue for that) so I replaced it with a custom solution.

_If someone is interested in my solution:_
I needed to display 2 lines of chips, each one containing a different text, so in the `build` function I compute how many chip I can display using the current `constraints` and a `TextPainter` to estimate every chip's size.
Then I place the sublist of children in a regular Wrap, with some other logic to add the overflowWidget and remove the last child if needed.
As I sayd this is NOT a generic approach.
